src/cli.js

```diff
@@ -32,6 +32,7 @@ program
     .option('-p, --privileged', 'Give extended privileges to the executed command process')
     .option('-v, --verbose', 'Adds additional logging')
     .option('-d, --detached', 'Detached mode: Run command in the background.')
+    .option('-l, --list', 'List the available actions that can be run as ' + chalk.gray('dopr <action>'))
```

Comment:
how about `-h, --help` instead? this would be more intuitive i believe

Reply:
hm, but that's already used by commander to list available options to dopr. (-p/-v/-d/-e etc and their description)